rocketpy/stochastic/stochastic_rocket.py

```diff
@@ -189,45 +163,6 @@ def __init__(
         self.rail_buttons = Components()
         self.parachutes = []
 
-    def __str__(self):
```

Comment:
Why is this not needed anymore?

Reply:
See PR #612, the `.visualize_attributes` replaced this feature.